app/views/users/new.html.haml

```diff
@@ -1,12 +1,38 @@
+<script type="text/javascript" src="https://js.stripe.com/v2/"></script>
+
+<script type="text/javascript">
+Stripe.setPublishableKey("#{ENV['STRIPE_PUBLISHABLE_KEY']}");
+</script>
+
+= javascript_include_tag 'payment'
+
 %section.register.container
   .row
     .col-sm-10.col-sm-offset-1
-      = bootstrap_form_for @user, html: {class: 'form-horizontal'} do |f|
+      = bootstrap_form_for @user, html: {class: 'form-horizontal', id: 'payment-form'} do |f|
```

Comment:
This is not the right way to do horizontal forms anymore. You need to use `layout: :horizontal`. https://github.com/bootstrap-ruby/rails-bootstrap-forms#horizontal-forms